synfig-core/src/synfig/layers/layer_shape.h

```diff
@@ -73,7 +73,7 @@ class Layer_Shape : public Layer_Composite, public Layer_NoDeform
 	Vector feather;
 
 	mutable Time last_sync_time;
-	mutable Real last_sync_outline_grow;
+	mutable Real last_sync_outline_grow = 0.l;
```

Comment:
Is this not zero to force the first `Layer::sync()`?

Reply:
> Is this not zero to force the first `Layer::sync()`?

```
Layer_Shape::sync(bool force) const
{
	if ( force
	  || !last_sync_time.is_equal(get_time_mark())
	  || fabs(last_sync_outline_grow - get_outline_grow_mark()) > 1e-8 )
```

I don't know. My idea is to initialize it with zero and fix if it breaks something.
At least this will be consistent result.